recipes/unleash-client-cpp/all/conanfile.py

```diff
@@ -42,16 +33,15 @@ def _min_cppstd(self):
     def _compilers_min_version(self):
         return {
             "Visual Studio": "15",  # Should we check toolset?
+            "msvc": "191",
             "gcc": "7",
             "clang": "4.0",
             "apple-clang": "3.8",
             "intel": "17",
```

Comment:
`intel` is only v1 - `intel-cc` is for v2, but the versions differ https://docs.conan.io/2/reference/config_files/settings.html (Something that https://github.com/conan-io/conan-center-index/pull/18739/files#diff-e17e1d2d544d2e48d2419318736b0ecbd6ac0e2c6f74b6936759e8358e289112R44 missed)